### PR TITLE
Migrate to Apache license v2

### LINKED ISSUE #158: Migrate to jaegertracing org
Per https://github.com/jaegertracing/jaeger/issues/409

  * [x] Move the repo
  * [x] Re-register the repo with Travis
  * [x] Re-register the repo with Coveralls/CodeCov
  * [x] Re-encrypt credentials
  * [x] Update the links to badges in the README
  * [x] Add DCO file
  * [x] Add sign-off checker bot
  * [x] Update CONTRIBUTING instructions to require sign-off
  * [x] Update to Apache license
    * [x] Replace LICENSE file
    * [x] Update CONTRIBUTING instructions and references to license in the README
    * [x] Update the existing license headers (keep Uber copyright)
    * [x] Update the check-license script (new files get The Jaeger Authors copyright)

### PR DESCRIPTION
**Pull request by objectiser**
Fixes #158 

Signed-off-by: Gary Brown <gary@brownuk.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -18,13 +16,103 @@ Before you get started, here are some suggestions:
  3. `npm test`
  4. `make build-node`
 
-## Licencing
+## Making A Change
 
- - Every file must have a licence block at the top. This is enforced using `uber-licence`
```
Comment (yurishkuro) [inline CONTRIBUTING.md]: What is the replacement for uber-license?  I don't see any changes to `package.json`, how did you perform replacement of the license headers? I think we need
  * a validation step that fails the build if files don't have a license header
  * optionally, an auto-formatter that adds the headers to new files
  *

Comment (objectiser) [reply CONTRIBUTING.md]: Will investigate to see what options are available.

Comment (objectiser) [reply CONTRIBUTING.md]: perl :) - had a look at uber-license by it only uses MIT license. Will have a look for an automated checker/formatter tomorrow - hopefully we can find something suitable.

Comment (yurishkuro) [reply CONTRIBUTING.md]: > perl

can you post it?  Might come in handy in other repos.

Comment (objectiser) [reply CONTRIBUTING.md]: Sure: https://gist.github.com/objectiser/81491af29de2d1043764425f9c7d90a6

**Pull request merged**